README.md

```diff
@@ -80,7 +80,6 @@ environment: production
 max_persistent_conns: 512
 servers: 1
 threaded: true
-no-epoll: true
```

Comment:
I removed this from the README because people mind blindly copy this, have no idea what epoll is and accidentally disable it.